core/src/main/java/fr/sncf/osrd/infra/implementation/tracks/undirected/TrackSectionImpl.java

```diff
@@ -16,6 +16,8 @@ public class TrackSectionImpl implements TrackSection {
     EnumMap<Direction, RangeMap<Double, SpeedLimits>> speedSections;
     RangeMap<Double, String> catenaryVoltages = TreeRangeMap.create();
     EnumMap<Direction, RangeMap<Double, Double>> gradients;
+    EnumMap<Direction, RangeMap<Double, Double>> curves;
+    EnumMap<Direction, RangeMap<Double, Double>> slopes;
```

Comment:
Do we still need to keep `gradients`? Slopes and curves were merged together to simplify the code, because we didn't need to handle them separately. Now that we do, it now adds complexity instead

Note: this file (and most of the parsing) has a good chance of being deprecated or rewritten for https://github.com/DGEXSolutions/osrd/issues/4064